src/java.desktop/share/classes/sun/java2d/cmm/lcms/LCMSTransform.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2007, 2016, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2007, 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Review took so long that years here and in other headers of modified files needs to be updated.

Reply:
Fixed.